vminitd/Sources/vmexec/RunCommand.swift

```diff
@@ -93,7 +98,30 @@ struct RunCommand: ParsableCommand {
                 throw App.Errno(stage: "setsid()")
             }
 
-            try childRootSetup(rootfs: root, mounts: spec.mounts, log: log)
+            try childRootSetup(rootfs: root, mounts: spec.mounts, process: process, log: log)
```

Comment:
Wait, how does this work? Inside `childRootSetup` we do the rootfs setup dance, and one of those steps includes `configureConsole`. However, we haven't set stdio to the pty we create here yet so fd 0 should still be /dev/null I'd imagine. I think we need to move `configureConsole` to inside of the below `if process.terminal` branch as the last step. Also we don't need to do the readlink dance anymore as we can just `ptsname()` now.